vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -231,7 +231,7 @@ bool AdsImpl::IsInitialized() {
 void AdsImpl::Shutdown(
     ShutdownCallback callback) {
   if (!is_initialized_) {
-    BLOG(WARNING) << "Failed to shutdown ads as not initialized";
+    BLOG(WARNING) << "Shutdown failed as not initialized";
```

Comment:
Better as (INFO)? It's fair to say you can't shut something down if it's not initialised, so really this is just diagnostic at this point to say something called it prematurely?

Reply:
Our logs are quite busy as is, so wanted to reduce noise. When diagnosing issues we use a higher verbosity.